### PR TITLE
[OpenAPI] Trim operation name prefixes

### PR DESCRIPTION
**Pull request by newtork**
## Context

AI/ai-sdk-java-backlog#53.

I've noticed the internal OpenAPI generator has means to remove operation name prefixes. This way the method names in API classes can be defused.

```xml
<additionalProperties>
  <removeOperationIdPrefix>true</removeOperationIdPrefix>
  <removeOperationIdPrefixDelimiter>\.</removeOperationIdPrefixDelimiter>
  <removeOperationIdPrefixCount>3</removeOperationIdPrefixCount>
</additionalProperties>
```

## Definition of Done

- [x] Functionality scope stated & covered
- [x] Tests cover the scope above
- [x] ~Error handling created / updated & covered by the tests above~
- [x] ~Aligned changes with the JavaScript SDK~
- [x] ~Documentation updated~
- [x] ~Release notes updated~

### CODE REVIEW

File: e2e-test-app/src/main/java/com/sap/ai/sdk/app/controllers/ScenarioController.java [hunk 1]
```diff
@@ -34,6 +34,6 @@ AiScenarioList getScenarios() {
   @GetMapping("/models")
   @Nullable
   public AiModelList getModels() {
-    return API.modelsGet("foundation-models", "default");
+    return API.get_0("foundation-models", "default");
   }
```
Comment (newtork) [inline e2e-test-app/src/main/java/com/sap/ai/sdk/app/controllers/ScenarioController.java:38]: (Major)

This is a problem.

Cause: Following operations resolve to the same method names + signature `(String, String)`:

```yaml
get:
  tags:
  - Scenario
  summary: Get information about all models available in LLM global scenario
  description: Retrieve information about all models available in LLM global scenario
  operationId: models.get
```

```yaml
get:
  tags:
  - Scenario
  summary: Get scenario by id
  description: Retrieve details for a scenario specified by scenarioId.
  operationId: scenario.get
```

[If it was fixed](https://github.wdf.sap.corp/AI/apis/blob/master/ai_api.yaml#L906), we could use the logic just fine:

 ```diff
- operationId: models.get
+ operationId: scenario.get_models
```

I created [a PR to address this inconsistency here](https://github.wdf.sap.corp/AI/ml-api-facade/pull/1242)

Comment (CharlesDuboisSAP) [reply e2e-test-app/src/main/java/com/sap/ai/sdk/app/controllers/ScenarioController.java]: The fix has been merged

**Review by CharlesDuboisSAP — APPROVED** [review]
@newtork please review my commits before merging

**Review by newtork — COMMENTED** [review]
LGTM

**Pull request merged**